app/assets/javascripts/components/event_form/stores/event_form_store.cjsx

```diff
@@ -22,7 +22,7 @@ class EventFormStore
       getServerErrors:  this.getServerErrors
 
   handleFormatMultiselect: ->
-    if this.formData["eventType"] != "3"
+    if this.formData["eventType"] != "3" and this.formData["collectiveMembersMulti"]
```

Comment:
actually I'm going to change this so the frontend also doesn't care about event type here (cuz it doesn't actually matter)